pkgs/top-level/python-packages.nix

```diff
@@ -12276,6 +12278,26 @@ in {
     };
   };
 
+  periodictable = buildPythonPackage rec{
+    name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.